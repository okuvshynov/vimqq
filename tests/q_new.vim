function! WriteAndQuit(t)
    :normal q
    execute "write q_new.out"
endfunction

function! AskNew(t)
    :Q -n @mock world!
    call timer_start(1000, "WriteAndQuit")
endfunction

:Q @mock hello
call timer_start(1000, "AskNew")

