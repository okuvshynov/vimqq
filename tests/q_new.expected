00:00>l=152
00:00 l=151
